bot/exts/halloween/candy_collection.py

```diff
@@ -197,20 +185,15 @@ async def candy(self, ctx: commands.Context) -> None:
             '\N{SPORTS MEDAL}'
         )
 
-        top_sorted = sorted(self.candy_json['records'], key=lambda k: k.get('record', 0), reverse=True)
+        top_sorted = sorted(((user_id, score) for user_id, score in self.candy_records.items()),
+                            key=lambda x: x[1], reverse=True)
         top_five = top_sorted[:5]
 
-        usersid = []
-        records = []
-        for record in top_five:
-            usersid.append(record['userid'])
-            records.append(record['record'])
-
-        value = '\n'.join(f'{emoji[index]} <@{usersid[index]}>: {records[index]}'
-                          for index in range(0, len(usersid))) or 'No Candies'
+        leaderboard = '\n'.join(f"{emoji[index]} <@{record[0]}>: {record[1]}"
+                                for index, record in enumerate(top_five)) or 'No Candies'
 
         e = discord.Embed(colour=discord.Colour.blurple())
-        e.add_field(name="Top Candy Records", value=value, inline=False)
+        e.add_field(name="Top Candy Records", value=leaderboard, inline=False)
         e.add_field(name='\u200b',
                     value="Candies will randomly appear on messages sent. "
                           "\nHit the candy when it appears as fast as possible to get the candy! "
```

Comment:
```suggestion
        e.add_field(
            name='\u200b',
            value="Candies will randomly appear on messages sent. "
                  "\nHit the candy when it appears as fast as possible to get the candy! "
                  "\nBut beware the ghosts...",
            inline=False
        )
```
better indentation for the whole add_field()